docs/source/commands/graphs.mdx

```diff
@@ -130,6 +130,64 @@ If you don't want to wait for the check to complete, you can run the command wit
 
 If you're running schema checks in CI, you might want to pass the `--background` flag to `rover graph check`. This flag instructs Rover to initiate schema checks but _not_ await their result. If you've [connected GraphOS to your GitHub repository](/graphos/delivery/github-integration/), the integration detects the checks execution and adds a status to the associated pull request.
 
+### `graph lint`
+
+<AuthNotice />
```

Comment:
This notice declares that the command requires Rover to be authenticated with GraphOS. Is that always the case for these commands?

Reply:
Yes, that is the intention. You should be authenticated with GraphOS in order to run lints for both graph and subgraph